config/routes.rb

```diff
@@ -6,6 +6,7 @@
   resources :loan_types
   resources :documents
   resources :loans
+  resources :loan_documents
```

Comment:
@ericksonmc except show, new/create and delete. this is far less than all the resources.